pkg/kubelet/config/common.go

```diff
@@ -43,6 +43,10 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	maxReadLength = 10 * 1 << 20 // 10MB
```

Comment:
nit: can you be a bit more descriptive with these variable names? i.e. what is being read?

In this case, maybe "maxManifestFileSize", but please update all instances of `maxReadLength`

Reply:
Done